hack/dind-cluster.sh

```diff
@@ -460,6 +460,8 @@ function get-network-plugin() {
     echo "${ovn_plugin}"
   elif [[ "${plugin}" = "cni" ]]; then
     echo "cni"
+  elif [[ "${plugin}" = "none" ]]; then
+    echo ""
```

Comment:
This works but seems odd way of handling command line options.
Why not make '-n' param optional?  If -n is passed then it has to be valid plugin and if this flag is not passed then it means don't use any network plugin.

Reply:
Since dind is only used for development environment, this workaround may be okay.

Reply:
Because the existing behavior defaults to a plugin :-(